tests/periph_gpio_no_stdio/Makefile

```diff
@@ -0,0 +1,9 @@
+include ../Makefile.tests_common
```

Comment:
Probably this will need to add an `app.config.test` to enable `CONFIG_MODULE_PERIPH_GPIO` and `CONFIG_MODULE_STDIO_NULL`, and a `Kconfig` to `imply MODULE_PERIPH_GPIO_IRQ`.

Reply:
Thanks. Murdock is happy, so I guess this works :-)